src/org/usfirst/frc/team694/robot/subsystems/Acquirer.java

```diff
@@ -43,10 +48,19 @@ public void deacquire() {
     }
 
     public void flipUp() {
-        acquirerSolenoid.set(true);
+        acquirerFlipSolenoid.set(true);
     }
 
     public void flipDown() {
-        acquirerSolenoid.set(false);
+        acquirerFlipSolenoid.set(false);
+        Timer.delay(0.5);
+    }
+    
+    public void holdCube() {
```

Comment:
The acquirer solenoid does not "hold" or "release" the cube, it tightens and loosens its grip around the cube (which is already held). I'd suggest renaming this to reflect that